packages/react-component-library/src/components/Timeline/Timeline.tsx

```diff
@@ -73,16 +73,13 @@ function extractRowData(
 ) {
   return (rowGroups as []).map(
     ({ props: { children } }: timelineBodyChildrenType) => {
-      let rows: any[] = []
-
-      if (children) {
-        const items = Array.isArray(children) ? children : [children]
-        rows = items.map(
+      const rows =
+        React.Children.map(
+          children,
           ({ props: { name } }: React.ReactElement<TimelineRowProps>) => ({
             name,
           })
-        )
-      }
+        ) || []
```

Comment:
If I remove `|| []` then I cannot see anything breaking.

Reply:
Not returning the default seems to break some of the storybook stories that include sidebar but no rows.

<img width="1495" alt="Screenshot 2020-07-07 at 13 58 09" src="https://user-images.githubusercontent.com/48086589/86784947-03b4ed80-c05a-11ea-9764-b2c3238dbc0e.png">

Reply:
https://github.com/Royal-Navy/design-system/blob/e572fbece322bed8c7f4483a0956a98054078d01/packages/react-component-library/src/components/Timeline/TimelineSide.tsx#L35

Reply:
Okay, the overarching point is that unit tests do not fail either. I think if we were testing `TimelineSide` from the outside `Timeline` then there would not be this gap.

Reply:
I've added some additional tests to cover TimelineSide.